### DISCUSSION TITLE
Disable keyboard shortcuts without disabling snap areas

### DISCUSSION THREAD

**Discussion by nchammas** [rxhanson/Rectangle#1283] [Ideas]
Is there a way to disable keyboard shortcuts without disabling the snap areas?

I ask because I rarely use the keyboard shortcuts, yet they sometimes interfere with apps I'm using.

For example, Databricks, which has a browser-based notebook UI similar to [Jupyter][1], has many shortcuts for manipulating cells within a notebook. One of the shortcuts that conflicts with Rectangle, for example, is the shortcut to split a cell: `control + option + -`

I know I can change the conflicting shortcut in Rectangle, but there are many others that conflict too.

And I know I can set Rectangle to ignore Safari, but that also disables the snap areas too, which I don't want.

I want the snap areas of Rectangle without the keyboard shortcuts. I'd be fine with that being a global setting, as opposed to a per-app setting.

Is something like this possible? Is there a different way to think about this problem?

[1]: https://jupyter.org

**Comment by rxhanson** [comment]
Is this what you're looking for: https://github.com/rxhanson/Rectangle/discussions/1266

**Reply by nchammas** [reply]
Yes indeedy. Sorry, it didn't turn up for me when I did a search.
